spec/factories/checkin_times.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :checkin_time do
+    weekday 0
+    time    '9:00 a.m.'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.